components/TextFields/src/private/MDCTextInputCommonFundament.m

```diff
@@ -384,14 +388,16 @@ - (void)setupUnderlineLabels {
   _trailingUnderlineLabel.opaque = NO;
   [_textInput addSubview:_trailingUnderlineLabel];
 
+  UILayoutPriority defaultPriority = UILayoutPriorityDefaultHigh;
```

Comment:
We need to find a way to do this with constraints priorities below 260 in the input.

Reply:
@willlarche, we prefer lower priorities because we need the controller to be able to override the constraints, correct? Can't a controller still override a constraint with a priority of 750 if it needs to?

It just seems to me that these constraints, which are there to keep the labels' contents inside the view bounds, will fail to do so if their priorities are too low and the text grows too long. That's actually what I would _expect_ to happen... 

When would a controller want to overpower these constraints, which seem to me to serve a vital purpose?
Would you be happier with an approach that adds fixed width constraints to the labels? I haven't tried it yet, but it might work as well...

Thank you for your attention/assistance!